### ISSUE TITLE
Windows: Access Violation when calling executemany()

### ISSUE THREAD

**Issue by deymundson** [oracle/python-oracledb#217]
1. What versions are you using?
Database Version: 19.14.0.0.0

platform.platform: Windows-10-10.0.19045-SP0
sys.maxsize > 2**32: True
platform.python_version: 3.11.4
oracledb.__version__: 1.4.0

2. Is it an error or a hang or a crash?
Crash

3. What error(s) or behavior you are seeing?
The python program immediately exits. The last exit code is `-1073741819`, which seems to indicate some form of access violation.
```
> .\python.exe
Python 3.11.4 (tags/v3.11.4:d2340ef, Jun  7 2023, 05:45:37) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import oracledb
>>> oracledb.init_oracle_client()
>>> connection = oracledb.connect([REDACTED] ;P)
>>> cursor = connection.cursor()
>>> cursor.setinputsizes(None, int)
[None, <oracledb.Var of type DB_TYPE_NUMBER with value None>]
>>> cursor.executemany("select :1, :2 from dual", [])
[[Program immediately exits]]
> $LASTEXITCODE
-1073741819
```

4. Does your application call init_oracle_client()?
Yes

5. Include a runnable Python script that shows the problem.
```Python
import oracledb
oracledb.init_oracle_client()
connection = oracledb.connect(...)
cursor = connection.cursor()
cursor.setinputsizes(None, int)
cursor.executemany("select :1, :2 from dual", [])
```

**Comment by cjbj** [comment]
Thanks for the report.

**Comment by anthony-tuininga** [comment]
I have pushed a patch that should correct this issue. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it corrects your issue as well.

**Comment by anthony-tuininga** [comment]
This was included in version 1.4.1 which was just released.
